### DISCUSSION TITLE
Report protection !!

### DISCUSSION THREAD

**Discussion by nastimch** [YimMenu/YimMenu#3446] [General]
is Yimmenu have protection reporting players if yes how can i activate it because i got suspend my account for 30 days cause report player on public session
when i did teleport to him and kill him !!

**Comment by Yimura** [comment]
Report protection doesn't exist anymore since Rockstar made some changes to how that works. As such it's best to just behave in online.

Long story short: in the past reports would be sent to your game, these reports could be intercepted and blocked. Rockstar made a change that if a player reports you the report is sent directly to Rockstar.

**Comment by nastimch** [comment]
but i see kiddons menu have protection for it !!

**Reply by Yimura** [reply]
Is Kiddions modest menu up-to-date for the most recent update? Anyways he can have that feature it’s not gonna change anything. We can’t block something that’s never sent to us in the first place. As I’ve said in the other comment.

**Comment by ghost** [comment]
> is Yimmenu have protection reporting players if yes how can i activate it because i got suspend my account for 30 days cause report player on public session when i did teleport to him and kill him !!

I reported you and you got banned HAHAHIHI

**Reply by nastimch** [reply]
really ? 
what was selling !!

**Reply by ghost** [reply]
> what was selling !!

Wdym? 
I wanted to say that reports don't work. I had about 500 reports half a year ago and I'm still not banned

**Comment by snowleopardd** [comment]
Time to report all the p2c users hahaha

**Reply by ghost** [reply]
it's so funny to watch this guy flood every discussion that he got banned at the same time he knows that kiddions has protection from reports. of course if you try different menus then 99% of you will get banned in a month. especially considering the fact that kiddions is updated once a century

**Reply by snowleopardd** [reply]
> it's so funny to watch this guy flood every discussion that he got banned at the same time he knows that kiddions has protection from reports. of course if you try different menus then 99% of you will get banned in a month. especially considering the fact that kiddions is updated once a century

yeah also i'm pretty sure he used many recovery options, rule of thumb to not get banned is to not be a dumbass and don't do money shit, unless it's a super safe method like the bunker one and you really need the money for something that u can't simply spawn

**Reply by nastimch** [reply]
> t sim

what u mean with recovery money ! is sell bunker bussines with teleport !! if yes many people using mod menu for help sell staff no one get ban I got ban from reporting player 95% with my fault using mod on public session and kill rivals that all

**Reply by ghost** [reply]
> > t sim
> 
> what u mean with recovery money ! is sell bunker bussines with teleport !! if yes many people using mod menu for help sell staff no one get ban I got ban from reporting player 95% with my fault using mod on public session and kill rivals that all

what mods have you used besides yim and for how long?

**Reply by nastimch** [reply]
only yimenu and only teleport and sometimes god mod that all also no scripts uploads to yimenu !!
